app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="ems_value">5</integer>
+    <integer name="max_length">18</integer>
+</resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done